src/main/java/com/keeper/homepage/domain/game/entity/embedded/Baseball.java

```diff
@@ -0,0 +1,56 @@
+package com.keeper.homepage.domain.game.entity.embedded;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Embeddable;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Builder
+@AllArgsConstructor(access = AccessLevel.PUBLIC)
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Embeddable
+public class Baseball {
+
+  private static final int BASEBALL_MAX_PLAYTIME = 1;
+
+  @Column(name = "baseball_per_day", nullable = false)
+  private Integer baseballPerDay;
+
+  @Column(name = "baseball_day_point", nullable = false)
+  private Integer baseballDayPoint;
+
+  public static Baseball from(Integer baseballPerDay, Integer baseballDayPoint) {
+    return new Baseball(baseballPerDay, baseballPerDay);
+  }
```

Comment:
```suggestion
  public static Baseball from(Integer baseballPerDay, Integer baseballDayPoint) {
    return new Baseball(baseballPerDay, baseballDayPoint);
  }
```

이렇게 수정해야 할 것 같습니다 ㅎㅎ

Reply:
헐 최고